submissions/satisfactorio/satisfactorio.md

```diff
@@ -0,0 +1,91 @@
+## Welcome to Satisfactorio!
+
+### What?
+
+**Satisfactorio** is an overhaul mod that bridges the gap between Satisfactory and Factorio. Every feature, from Milestones to Drones and everything in between, has been carefully re-created in Factorio.
+
+### Why?
+
+Modding isn't about "why", it's about "why not"! I enjoyed both Factorio and Satisfactory, and over on the Satisfactory mod portal there's a mod that attempts to bring Factorio elements into Satisfactory. Things like Inserters, Science Packs and other small scale modifications. That seemed like a cool concept so I wondered if maybe I could bring Satisfactory elements into Factorio. This [has been attempted](https://mods.factorio.com/mod/Satisfactory) before, however development doesn't seem to have gone particularly far before it was abandoned.
+
+In particular I was interested in how Satisfactory conveyor belts connect directly to input/output slots on machines. This also applies to train stations, where a train pulls in and dumps its cargo in the platform storage, to then be loaded onto a belt. Certainly a lot simpler than some of the optimised loading/unloading setups I've seen in Factorio!
```

Comment:
You could also mention here that Satisfactory had single lanes? If that was part of your thought process. 

I also think they used direct connections (and not inserters) for performance reasons because it's _muuuuch_ less calculations. IIRC the Technical Factorio guys always say inserters are very UPS expensive. But that's just me rambling :P